wurst/_items/ScrollOfCyclone.wurst

```diff
@@ -0,0 +1,20 @@
+package ScrollOfCyclone
+
+import ClosureEvents
+import ClosureTimers
+
+let ABILITY_SCROLL_OF_CYCLONE = 'S000'
+let BUFF_CYCLONE = 'Bcyc'
+let BUFF_CYCLONE_EXTRA = 'Bcy2'
+
+function onCyclone()
+    let target = GetSpellTargetUnit()
+    target.addAbility(GHOST_VIS_ID)
+    doPeriodicallyTimed(0.1, 15) buffPoller ->
```

Comment:
You can't end cyclone early such as using a cure. So you could just use `doAfter`.